wiki/Rules/zh.md

```diff
@@ -1,59 +1,56 @@
-6. **规定中没有指出的，请根据常识做出判断。** 管理员有权利给出他们认为合适的裁决。
+6. **规定中没有指出的，请根据常识做出判断。** 管理员有权给出他们认为合适的裁决。
 
 ## 游戏聊天频道规定
 
 1. **尊重聊天频道内的所有玩家，不要刷屏。** 如果你的聊天占据了半个屏幕，别人就不能愉快的聊天了。
-2. **让频道管理员处理聊天内的问题。** 使用`!report`命令，他们收到后会进行处理。
+2. **让频道管理员处理聊天内的问题。** 使用[`!report`命令](/wiki/Reporting_Bad_Behaviour)，他们收到后会进行处理。
 3. **不要讨论被禁言或限制的玩家。** 他的问题只能由他自己和管理员解决，而不是其他人。
-4. **鼓励公平竞争，不要作弊。** 如果你发现了作弊者，请使用`!report`命令举报，或是在他的个人资料页点击举报。
-5. **语言得体，避免 R18/NSFW 内容。** 公共聊天频道不能发布这些内容，但是允许在私聊中发。
+4. **鼓励公平竞争，不要关注作弊玩家。** 很遗憾，作弊玩家确实存在，给予他们过多关注会鼓励他们继续或吸引他人模仿。请使用`!report`命令或是在个人资料页点击来举报他们。
+5. **语言得体，适合全年龄，避免 R18/NSFW 内容。** 公共聊天频道不能发布这些内容，但是允许在私聊中发。
 6. **注意你与他人分享的内容，尤其是可能导致不适的内容。** 分享之前三思而后行。
-7. **确保链接符合法律规定。** 盗版链接，以及“[warez](http://baike.baidu.com/item/WareZ)”相关的内容都不能出现在聊天频道。
-8. **请勿发布第三方聊天工具的链接，包括 Discord，Skype，QQ。** 如果你需要邀请某人加入你的聊天服务器，请在私聊中发。
-9. **在指定了语言的聊天频道，必须使用该语言。** 在没有指定语言的聊天频道，一律使用英文。但是`#lobby`和`#help`允许使用任何语言。*频道管理员在进行工作时可以忽略这条规定。*
+7. **保持链接中的网站相关且合法。** 盗版视频（不包括具有官方授权的视频网站），盗版软件，以及“[warez](http://baike.baidu.com/item/WareZ)”相关的内容都不能出现在聊天频道。可以给游戏直播打广告，但只能在 `#videogames` 频道。
+8. **请勿发布第三方聊天工具的链接，包括 Discord，Skype，QQ。** osu! 聊天频道是关于 osu! 本身的，而不是你的聊天服务器。如果你需要邀请某人，请在私聊中发。
+9. **在指定了语言的聊天频道，必须使用该语言（比如在 `#english` 说英文，`#chinese` 说中文）。** 在没有指定语言的聊天频道，一律使用英文。但是 `#lobby` 和 `#help` 允许使用任何语言。*[频道管理员](/wiki/People/Global_Moderation_Team)在进行工作时可以忽略这条规定。*
 
 ## 论坛规定
 
-1. **只能发布有意思的主题。** 论坛的重点在于讨论，所以确保你的主题值得大家一起讨论。
-2. **发言时注意内容必须全年龄适宜。**
+1. **只能发布有意义或有意思的主题。** 论坛的重点在于讨论，所以确保你的主题值得大家一起讨论。
+2. **发言时注意内容必须全年龄适宜。** NSFW 内容不适合全年龄社区，不过在私聊中随意。
 3. **批评别人时就事论事，请勿进行人身攻击。** 有时候适当的批评能给对方激励，但是人身攻击不是。
-4. **请勿挖坟。** 如果一个主题长时间无人回复，就让它被掩埋吧。
-5. **确保签名和头像符合规范。** 如果有动图，则它们不能包含高速的闪光，尺寸应在500KB以下。
-6. **请勿发布广告，推荐链接等其他令人不满的内容。** 自动播放的音乐、视频也属于这一范畴。
-7. **请勿发布未知风险的应用程序。**
+4. **请勿挖坟。** 如果一个主题长时间无人回复，就让它被掩埋吧。开一个新主题，或者搜索一下有没有人已经回答了你的问题。
+5. **确保签名和头像符合规范。** 如果有动图，则它们不能包含高速的闪光。图像应当具有小尺寸，以在低速网络下不至于缺失，最好在 500KB 以下。
+6. **请勿发布广告，推荐链接等其他令人不满的内容。** 邀请链接对发布者是有趣的，但其他人会感到厌烦，所以不要发。自动播放的音乐、视频也属于这一范畴。
+7. **请勿发布未知风险的应用程序。** 简单来说，如果对所谓工具的合法性有任何疑问，就不该发。
 
 ## 谱面讨论帖规定
 
-这些规定适用于谱面和它的讨论帖，它们均是通过游戏内[编辑器](/wiki/editor)的[谱面提交系统](/wiki/Glossary#bss)提交的。
+*注：这些规定适用于**所有被上传的谱面**，而不仅仅是 Ranked 的那些。对于 Ranked 谱面的更详细的要求，参考[Ranking Criteria](/wiki/Ranking_Criteria)*
 
-更详细的要求，参考[Ranked指南](/wiki/Ranking_Criteria)。
+这些规定适用于通过游戏内的[谱面提交系统](/wiki/Glossary#bss)提交的谱面。当 osu! 客户端在[编辑器](/wiki/editor)中提交谱面的适合会创建相应的讨论贴。
 
-1. **确保你拥有谱面内所有内容的使用权。** 包括歌曲、视频、击打声、图片以及其它不是你原创的内容。如果你不确定，我们为你提供了一个库，你可以在[杰出艺术家](https://osu.ppy.sh/beatmaps/artists/)列表中找到这个库，它们可以在 osu! 中免费使用。
+1. **确保你拥有谱面内所有内容的使用权。** 包括歌曲、视频、击打声、图片以及其它不是你原创的内容。如果你不确定，我们为你提供了一个不断增长的[精选艺术家](https://osu.ppy.sh/beatmaps/artists/)曲目库，它们全都被授权在 osu! 中使用。
 2. **谱面讨论帖内不要脱离主题。** 这些帖子是专门用来讨论谱面内的问题的。如果你有其它不相关的问题，请在合适的板块中提出。
-3. **不要抄袭他人的成果。** 你不应当在未被授权的情况下上传他人的成果。
+3. **不要抄袭他人的成果。** 你不应当在未被授权的情况下上传他人的成果，包括但不限于皮肤和客串谱面。
+4. **遵守[歌曲内容规定](Song_Content_Rules)和[视觉内容注意事项](Visual_Content_Considerations)。**
 
 ## 当我违反规定时会发生什么？
 
```

Comment:
missing something from line 49 in the English version

Reply:
It's somehow natural to combine line 49 and 51 into one sentence in Chinese. But splitting to achieve line to line matching is also OK.